components/post_view/channel_intro_message/channel_intro_message.jsx

```diff
@@ -104,7 +101,7 @@ function createGMIntroMessage(channel, centeredIntro, profiles) {
                         id='intro_messages.GM'
                         defaultMessage='This is the start of your group message history with {names}.\nMessages and files shared here are not shown to people outside this area.'
                         values={{
-                            names,
+                            names: channel.display_name,
```

Comment:
We can just the channel display name as it's already pre-populated with the correct names